cookbook/persistance/key-value.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: page
+title: "Storing key-value data on disk"
+permalink: /cookbook/persistence/key-value/
+---
+
+If you have a relatively small collection of key-values that you'd like to save, you should use the [shared_preferences](https://pub.dartlang.org/packages/shared_preferences) APIs
+
+Normally we would have to write native platform integration for storing data for both platforms, but fortunetly [shared_preferences](https://pub.dartlang.org/packages/shared_preferences) plugin can be used for that. 
+
+Shared preferences plugin wraps `NSUserDefaults` (on iOS) and `SharedPreferences` (on Android), providing a persistent store for simple data. Data is persisted to disk automatically and asynchronously.
```

Comment:
"The shared_preferences plugin wraps `NSUserDefaults` on iOS and `SharedPreferences` on Android" -- no need for the extra parens `()` 

Hrm, I find the line "Data is persisted to disk automatically" a bit confusing. Don't know if that sentence is necessary! We're handing the plugin some data manually, so  "automatic" sounds a bit funny.

Overall, I'd consider combining this part with the previous sentence to form a complete paragraph.